storage/client_status_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2016 The Cockroach Authors.
```

Comment:
Is this file's filename an error? "status_test" --> "stats_test" I think, unless I'm missing something.